examples/vec/src/main.sw

```diff
@@ -25,7 +25,8 @@ fn main() {
     // ANCHOR_END: vec_get_oob
     // ANCHOR: vec_iterate
     let mut i = 0;
-    while i < v.len() {
+    while i < v
+        .len() {
```

Comment:
yeah this is very strange to see, we should keep this rendering a `v.len()`. having len() on a new line isn't a good look.

Reply:
Any better now @JoshuaBatty ? I added a fix in e656a05d7